src/Wallet.tsx

```diff
@@ -48,6 +48,9 @@ const Wallet: React.FunctionComponent<{ role: Role }> = (props: {
       mySigningKey = import.meta.env.VITE_ALICE_SK;
       // @ts-expect-error
       myScwAddress = import.meta.env.VITE_ALICE_SCW_ADDRESS;
+      // @ts-expect-error
+      myPeerSCWAddress = import.meta.env.VITE_BOB_SCW_ADDRESS;
```

Comment:
We want to send payments to our peers SCW,  since that's what we're monitoring for the balance.

Reply:
I think at a point the `OwnerAddress`s were used because the SCW addresses weren't yet known.